tools/run_tests/helper_scripts/run_python.sh

```diff
@@ -18,7 +18,7 @@ set -ex
 # change to grpc repo root
 cd "$(dirname "$0")/../../.."
 
-PYTHON=$(realpath "${1:-py27/bin/python}")
+PYTHON=$(realpath "${1:-py35/bin/python}")
```

Comment:
Should we use `py37` if we can?

Reply:
Not many distros ship with 3.7 by default yet. 3.5 is the lowest common denominator for Python 3.

Reply:
I know it is a lot of trouble to test against those distributions.

My suggestion is use `py37` as our new default. If certain distribution doesn't have the `3.7`, we can always set it back to `3.5`.

Or, in majority of the use cases, we will set the `PYTHON` variable. For those we didn't set, probably we don't care about which version they run. Should we just point it to `python3` instead?

Reply:
Not even glinux ships with 3.7 by default yet.

Reply:
I would guess the default is never used in your testing infrastructure. So, I think it is okay to set an arbitrary Python version you see fit.